src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -221,6 +221,12 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NativeSystemLibrary Include="log" Condition="'$(_linuxLibcFlavor)' == 'bionic'" />
       <NativeSystemLibrary Include="icucore" Condition="'$(_IsApplePlatform)' == 'true'" />
       <NativeSystemLibrary Include="m" />
+      <!--
+        TODO: Double-check if libatomic's emulated CAS-128 works as expected once AOT applications are
```

Comment:
Nit: https://github.com/dotnet/runtime/blob/16b17d80869594e31398d5f01ecd320f263187a6/src/coreclr/nativeaot/Runtime/unix/PalRedhawkInline.h#L101 may be a better place for this TODO - to have all related TODOs in one place.